### ISSUE TITLE
Migrate to mousetrap for keyboard events

### ISSUE THREAD

**Issue by burtonator** [burtonator/polar-bookshelf#344]
https://github.com/ccampbell/mousetrap

**Comment by sotte** [comment]
I just want to chime in and say that better keyboard support would be nice!

Eg:
- arrow keys / hjkl for navigation between documents
- enter to open the document
- Ctrl-F or "/" for opening the search
- "t" to set tags

**Comment by burtonator** [comment]
agreed on the keyboard nav.. I think it will be control+enter for opening the document.  I think enter might be used to set the title. Not sure yet... baby steps.
